module/output/ImageCompressor/src/compressor/CompressorFactory.hpp

```diff
@@ -10,7 +10,7 @@ namespace module::output::compressor {
 
 class CompressorFactory {
 public:
-    virtual std::shared_ptr<Compressor> make_compressor(const uint32_t width,
+    virtual std::shared_ptr<Compressor> make_compressor(uint32_t width,
                                                         const uint32_t& height,
                                                         const uint32_t& format) = 0;
```

Comment:
It seems odd that width is passed by value.

Reply:
That does seem odd..., but the answer would be to make it a const reference, not remove the const

Reply:
I've had a look and it seems that width is immediately passed to a constructor that takes width as a const reference.